tests/dummy/app/components/events-example/component.js

```diff
@@ -1,9 +1,9 @@
 import Evented from '@ember/object/evented';
-import Controller from '@ember/controller';
+import Component from '@ember/component';
```

Comment:
I made this into a component and used `on('didInsertElement')` instead of `on('init')` in order to avoid cluttering up your example snippts with isFastBoot conditionals.

(Touching jQuery on init in fastboot blows up)